### DISCUSSION TITLE
Export from Wordpress into wiki.js

### DISCUSSION THREAD

**Discussion by iazz-code** [requarks/wiki#6402] [Help / Questions] [answered]
Hello,

we have a knowledgebase that was made in Wordpress years ago. We are now moving to wiki.js.

Has anyonve gone this way before? What's the best strategy? Right now I have no idea how to do that, because Wordpress just exports an XML-File that cannot be importet into wiki.js. If anoyone could point me into the right direction i would be pleased.

Wordpress 5.0.18 and wiki.js 2.5.298

Thanks
Thorsten

**Comment by fnafox** [comment, marked as answer]
Look into projects that offer exporting wordpress to markdown, there's a few around. You should then be able to import markdown pages into wiki.js without any issues under admin > storage > local file system.

**Reply by iazz-code** [reply]
Thanks a lot!

I used [wordpress-export-to-markdown](https://github.com/lonekorean/wordpress-export-to-markdown) and it worked out of the box. The only thing I had to do was to export our self signed certificate and point to it in a shell variable - but that was described in the issues-section.

Best
Thorsten
